src/components/ContentNav/__mocks__/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { SubItemProps } from '../SubItem';
+import { ItemProps } from '../Item';
+
+export const subMock1: SubItemProps = { name: 'sdgasdfasdf' };
+export const subMock2: SubItemProps = { name: 'helasd' };
+export const subMocks3: SubItemProps = [
+  {
+    active: false,
+    name: 'Task Descriptions'
+  },
+  {
+    active: false,
+    name: 'Target Evidence Statement'
+  },
+  {
+    active: false,
+    name: 'Appropriate Stems'
+  }
+];
+export const mock: ItemProps[] = [
```

Comment:
Same situation here. Be more specific; call this `itemMock`.

Reply:
It is renamed to itemMock.